lib/widgets/molecules/profile_page_social_chips.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:mm_flutter_app/widgets/atoms/social_chip.dart';
+
+class ProfilePageSocialChips extends StatelessWidget {
+  const ProfilePageSocialChips({Key? key}) : super(key: key);
+
+  List<Widget> _createChips() {
+    //TODO(guptarupal): Call backend instead of using hardcoded mock data.
+    return [
+      _createChip('LinkedIn', const Icon(Icons.person, color: Colors.white),
+          'https://www.linkedin.com/in/rupal-gupta-164821100/'),
+      _createChip('Facebook', const Icon(Icons.facebook, color: Colors.white),
+          'facebook_link'),
+      _createChip(
+          'Gmail', const Icon(Icons.email, color: Colors.white), 'gmail_link'),
```

Comment:
Please replace all hardcoded colors with colors from the color scheme. If the background color is 'primary' the icon on top should be 'onPrimary'. If the background color is 'surface', whatever you put on top should be 'onSurface'. That is pretty much how we ensure that the usage of color is consistent and meets minimum contrast requirements.

Read more about it here: https://m3.material.io/styles/color/the-color-system/color-roles#55d2b7d2-0202-4616-887e-f575a7946aac